src/components/dialogs/commons/text-field-input.tsx

```diff
@@ -0,0 +1,82 @@
+/**
+ * Copyright (c) 2023, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+import { FormattedMessage } from 'react-intl';
+import { TextField } from '@mui/material';
+import { makeStyles } from '@mui/styles';
+import React, { useCallback, useEffect } from 'react';
+
+const useStyles = makeStyles(() => ({
```

Comment:
We don't want to use makeStyles anymore, use [sx props](https://mui.com/system/getting-started/the-sx-prop/) instead

Reply:
done, thank you